### PR TITLE
Add helpbox

### LINKED ISSUE #352: Course Discovery Help Box
Lots of new incoming students are told to use AntAlmanac to plan their schedule but they have no clue what classes they are supposed to take.

On the Class Search page, we should have a help box that says something like "Don't know which classes to take? Look up your sample program" and link to the suggested classes page on the course catalogue. 

Here's an example for CS: https://catalogue.uci.edu/donaldbrenschoolofinformationandcomputersciences/departmentofcomputerscience/computerscience_bs/#sampleprogramtext

### PR DESCRIPTION
**Pull request by ChaseC99**
## Summary
Add a help box directing new students to the UCI Catalogue for help with class picking.
image

## Test Plan
- Verify that HelpBox looks good on desktop and mobile
- No typos
- Link to UCI Catalogue  

## Issues
Closes #352 

## Future Followup  
After enrollment, we could use the helpbox for other tips like highlighting AntAlmanac features (#303)

### CODE REVIEW

**Comment by EricPedley** [comment]
Maybe include a close button that remembers it's closed with localStorage, duplicate the info in the about page, and display a snackbar telling the user to look in the about page if they need to see it again.

**Comment by EricPedley** [comment]
spongebob burning eyes meme
Me looking at the light mode screenshot

**Comment by ChaseC99** [comment]
> Maybe include a close button that remembers it's closed with localStorage, duplicate the info in the about page, and display a snackbar telling the user to look in the about page if they need to see it again.

I thought about this but I wasn't sure if it was worth the effort, since we'll probably take it down once SPOP ends. Would you like me to do it though?

File: src/components/RightPane/CoursePane/SearchForm/HelpBox.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const HelpBox = (props) => {
+    const { classes } = props;
+    return (
+        <Paper variant="outlined" className={classes.container}>
+            <h2>Need help planning your schedule?</h2>
+            <ol className={classes.list}>
+                <li>
+                    Go to the{' '}
+                    <a href="https://catalogue.uci.edu/undergraduatedegrees/" target="_blank" rel="noopener noreferrer">
+                        UCI Catolouge
```
Comment (ChaseC99) [inline src/components/RightPane/CoursePane/SearchForm/HelpBox.js]:
```suggestion
                        UCI Catalogue
```

**Comment by ChaseC99** [comment]
> Me looking at the light mode screenshot

updated screenshot to darkmode 😌

**Comment by EricPedley** [comment]
> > Maybe include a close button that remembers it's closed with localStorage, duplicate the info in the about page, and display a snackbar telling the user to look in the about page if they need to see it again.
> 
> I thought about this but I wasn't sure if it was worth the effort, since we'll probably take it down once SPOP ends. Would you like me to do it though?

In that case, let's not add that functionality. Once we close this I'll create an issue to comment it out once SPOP is done.

**Review by EricPedley — APPROVED** [review]
LGTM 🚀

**Pull request merged**